smart_open/tests/test_s3.py

```diff
@@ -104,6 +104,26 @@ def ignore_resource_warnings():
     warnings.filterwarnings("ignore", category=ResourceWarning, message="unclosed.*<ssl.SSLSocket.*>")  # noqa
 
 
+class SeekableRawReaderTest(unittest.TestCase):
```

Comment:
Hide this test behind an environment variable (skip by default).

We shouldn't expect whoever is running the tests to also have moto on port 5000. It's good if they are, but it's not compulsory.

Reply:
Done. Please take a look.

Reply:
Very good. Thank you.